core/src/main/java/com/flipkart/gjex/core/web/filter/HttpAccessLogFilter.java

```diff
@@ -0,0 +1,54 @@
+package com.flipkart.gjex.core.web.filter;
+
+import com.flipkart.gjex.core.logging.Logging;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Filter for logging http access log requests
+ * @author ajay.jalgaonkar
+ *
+ */
+
+@Singleton
+@Named("HttpAccessLogFilter")
+public class HttpAccessLogFilter implements Filter, Logging {
+  private long startTime;
+  @Override
+  public void init(FilterConfig filterConfig) throws ServletException {}
+
+  @Override
+  public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
```

Comment:
We should change the signature to match how the grpc filter has

`public void doProcessRequest(R request) `

and 

`public void doProcessResponse(S response)`

Reply:
done